src/Stratis.Bitcoin.IntegrationTests/BlockStore/ReOrgRegularlySpecification_Steps.cs

```diff
@@ -0,0 +1,159 @@
+
+            this.sharedSteps.MineBlocks(10, this.selfishMiner, AccountZero, WalletZero, WalletPassword);
+
+            this.selfishBlockHeight = this.selfishMiner.FullNode.Chain.Height;
+        }
+
+        private void second_node_creates_a_transaction_and_broadcasts()
+        {
+            var thirdNodeReceivingAddress = this.thirdNode.FullNode.WalletManager()
+                .GetUnusedAddress(new WalletAccountReference(WalletZero, AccountZero));
+
+            this.secondNodeTransaction = this.secondNode.FullNode.WalletTransactionHandler().BuildTransaction(
+                SharedSteps.CreateTransactionBuildContext(new WalletAccountReference(WalletZero, AccountZero), WalletPassword, thirdNodeReceivingAddress.ScriptPubKey,
+                    Money.COIN * 1, FeeType.Medium, 101));
+
+            this.secondNode.FullNode.NodeService<WalletController>().SendTransaction(new SendTransactionRequest(this.secondNodeTransaction.ToHex()));
+        }
+
+        private void third_node_mines_this_block()
+        {
+            int feeForSecondToThirdTransaction = 100004520; /////?????
+            this.sharedSteps.MineBlocks(1, this.thirdNode, AccountZero, WalletZero, WalletPassword, feeForSecondToThirdTransaction);
+        }
+
+        private void fouth_node_confirms_it_ensures_tx_present()
+        {
+            this.sharedSteps.WaitForBlockStoreToSync(this.secondNode, this.thirdNode, this.fourthNode);
+
+            var transaction = this.fourthNode.FullNode.BlockStoreManager().BlockRepository.GetTrxAsync(this.secondNodeTransaction.GetHash()).Result;
+            transaction.Should().NotBeNull();
+            transaction.GetHash().Should().Be(this.secondNodeTransaction.GetHash());
+        }
+
+        private void selfish_node_reconnects_and_broadcasts()
+        {
+            this.selfishMiner.CreateRPCClient().AddNode(this.secondNode.Endpoint);
+            this.sharedSteps.WaitForBlockStoreToSync(this.selfishMiner, this.secondNode, this.thirdNode, this.fourthNode);
+        }
+
+        private void second_third_and_fourth_node_reorg_to_longest_chain()
+        {
+            TestHelper.WaitLoop(() => this.secondNode.FullNode.Chain.Height == this.selfishBlockHeight);
+            this.secondNode.FullNode.Chain.Height.Should().Be(this.selfishBlockHeight);
+
+            TestHelper.WaitLoop(() => this.thirdNode.FullNode.Chain.Height == this.selfishBlockHeight);
+            TestHelper.WaitLoop(() => this.fourthNode.FullNode.Chain.Height == this.selfishBlockHeight);
+        }
+
+        private void transaction_from_shorter_chain_is_missing()
+        {
+            var transaction = this.secondNode.FullNode.BlockStoreManager().BlockRepository.GetTrxAsync(this.secondNodeTransaction.GetHash()).Result;
+            transaction.Should().BeNull("longest chain comes from selfish miner and shouldn't contain the transaction made on the chain with the other 3 nodes.");
```

Comment:
Capital L

Reply:
failure message reads better with lowercase L as it is mid sentence due to Fluent assertions prefixing the `because` to form a proper sentence....

`Expected transaction to be <null> because longest chain comes from selfish miner and shouldn't contain the transaction made on the chain with the other 3 nodes..`